modules/logging/filters.go

```diff
@@ -25,6 +25,7 @@ import (
 
 func init() {
 	caddy.RegisterModule(DeleteFilter{})
+	caddy.RegisterModule(ReplaceFilter{})
```

Comment:
Let's hold off on this for now, we can do that in a separate PR. Keep this PR focused on just the global option, please!

Reply:
Sure, happy to do that 👍